tests/integration/test_interaction_assessment.py

```diff
@@ -220,15 +221,22 @@ def test_do_attempt_feedback_is_updated(self):
 
         self.click_submit()
 
+        # There are five items total (4 items with zones and one decoy item).
+        # We place the first item into correct zone and left the decoy item in the bank,
+        # which means the current grade is 2/5.
+        expected_grade = 2.0 / 5.0
+
         feedback_lines = [
             "FEEDBACK",
             FeedbackMessages.correctly_placed(1),
             FeedbackMessages.not_placed(3),
-            START_FEEDBACK
+            START_FEEDBACK,
+            FeedbackMessages.GRADE_FEEDBACK_TPL.format(score=expected_grade)
         ]
         expected_feedback = "\n".join(feedback_lines)
         self.assertEqual(self._get_feedback().text, expected_feedback)
 
+        # Place the item into incorrect zone. The score does not change.
```

Comment:
Is this comment true?  I am seeing different behavior as I mentioned in a previous comment.